includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -627,7 +610,7 @@ private function update_is_wcpay_subscriptions_enabled( WP_REST_Request $request
 	 */
 	private function update_account( WP_REST_Request $request ) {
 		$updated_fields_callback = function ( $value, string $key ) {
-			return in_array( $key, static::ACCOUNT_FIELDS_TO_UPDATE, true ) &&
+			return in_array( $key, array_keys( WC_Payment_Gateway_WCPay::ACCOUNT_SETTINGS_MAPPING ), true ) &&
```

Comment:
Can you maybe use[array_key_exists](https://www.php.net/manual/en/function.array-key-exists.php) here?

Reply:
Really good point! Addressed 9c2ecbe91e721026d9e28aace54c23b598473214